index.html

```diff
@@ -4,9 +4,66 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css"/>
+    <link rel="stylesheet" href="https://dazzle-ui-components.netlify.app/components/components.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"/>
     <title>Dazzle UI</title>
 </head>
 <body>
-    
+    <div class="index-main">
+        <header>
+            <div class="nav-section"></div>
+            <div class="nav-section">
```

Comment:
you have created this `<div class="nav-section"></div>` empty div to just use  `justify-content: space-between;`, is it? 

if that's the case, you can use ` justify-content: end;` on the flex container or `margin-left: auto` on the flex item.

Reply:
Ohh, I had an icon in the first section before. Decided to remove it, but forgot to change this. Thanks for pointing it out. I have made the changes.